readme.md

```diff
@@ -204,3 +205,32 @@ _Note: The realtime functionality counts against your Talk2M account's monthly b
 Tags can be deleted by right clicking the tag in the `Tag Browser` then selecting `Delete`. If the tag still exists on the Ewon or in Datamailbox the tag will be recreated in Ignition on the next Datamailbox update.
 
 <sup>[Back to top](#table-of-contents)</sup>
+
+## [Troubleshooting](#table-of-contents)
+
+### Troubleshooting Topics
+
+* [Tags do not appear in Ignition](#Tags-do-not-appear-in-Ignition)
+* [Changes on the Ewon Connector page do not take effect](#Changes-on-the-Ewon-Connector-page-do-not-take-effect)
+
+### Tags do not appear in Ignition
+
+There are several reasons tags may not be populated into Ignition. Please follow the following steps in order.
+
+1. Verify all steps in the [Ewon Setup](#ewon-setup) section were followed.
+2. If the system has just been configured wait one hour for data to be registered in Datamailbox.
+3. Verify your Talk2M credentials are correctly entered into the Ewon Connector settings in Ignition.
+   * If your credentials were incorrectly entered the Ignition logs will indicate a Authentication Error.  Check the logs on the Ignition webserver by clicking `Status` > `Logs`. If your credentials are incorrect you will see log entries in this format.
```

Comment:
comma between "entered" and "the Ignition logs" and between "incorrect" and "you"